modules/permission-sets/variables.tf

```diff
@@ -1,16 +1,16 @@
 variable "permission_sets" {
   type = list(object({
     name               = string
-    description        = string
-    relay_state        = string
-    session_duration   = string
-    tags               = map(string)
-    inline_policy      = string
-    policy_attachments = list(string)
-    customer_managed_policy_attachments = list(object({
+    description        = optional(string, "")
+    relay_state        = optional(string, "")
+    session_duration   = optional(string, "PT8H")
+    tags               = optional(map(string), {})
+    inline_policy      = optional(string, "")
+    policy_attachments = optional(list(string), [])
+    customer_managed_policy_attachments = optional(list(object({
       name = string
       path = string
-    }))
+    })), [])
```

Comment:
What does this do, exactly? I think this would be a validation error as this is not valid TF code.